.github/CODEOWNERS

```
@@ -7,6 +7,7 @@
 * @prometheus-community/helm-charts-admins
 
 /charts/alertmanager/ @monotek @naseemkullah
+/charts/jiralert/ @jkroepke
```

More than one maintainer would be better.
```suggestion
/charts/jiralert/ @jkroepke @zanhsieh 
```